crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -3060,6 +3082,33 @@ xGD..D..1..E..EGx
 xxxxxxxxxxxxxxxxx
 ENDMAP
 
+# Another NetHack-inspired vault, based on the top floor of Vlad's Tower.
+NAME:  nicolae_nh_vlads_tower
+TAGS:  transparent patrolling
+DEPTH: D:10-, Crypt
```

Comment:
Unless you want to add overflow tagging and move this to altars (which IMO it shouldn't be), start the depth D:11-, altar placement in D:3-10 should only happen by overflow temples.